pkg/skopeo/client.go

```diff
@@ -86,14 +86,12 @@ func (c *client) Inspect(registry, image string) (*DockerImage, error) {
 	}
 
 	if len(c.dockerAuthConfigs) != 0 {
+		// TODO: Is it a good idea to randomly select a secret?
 		i := rand.Intn(len(c.dockerAuthConfigs))
 		args = append(args, "--creds", c.dockerAuthConfigs[i].Username+":"+c.dockerAuthConfigs[i].Password)
 	}
 
 	config := "docker://" + image
-	if registry != "" {
```

Comment:
where is registry used, if it's removed there?

Reply:
now nowhere

Reply:
guess, it's wrong, users asked us about usage of private registries, if it's provided

Reply:
I removed the append because first, we never set the `registry` field so it was always `""`, and the registry part is already contained in the `image` param, so it just doubled and made it incorrect

Reply:
you mean, we don't set in our clustres? but our users use registry for long time. 
if we have it already in the image name, we can leave it empty or check, if it's already embedded and skip adding